cmd/sys-mgmt-executor/main.go

```diff
@@ -120,26 +128,36 @@ func checkDockerContainerStatus(service string, running bool) (bool, error) {
 	}
 }
 
-func ExecuteDockerCommands(service string, operation string) error {
+func ExecuteDockerCommands(service string, operation string) (string, error) {
 	// don't run commands for unknown services - could be an attack
 	if agent.IsKnownServiceKey(service) {
-		return runDockerCommands(service, operation)
+
+		LoggingClient.Info(fmt.Sprintf("4. IsKnownServiceKey() check a success for service %s", service))
+		LoggingClient.Info(fmt.Sprintf("5. operation: %s", operation))
```

Comment:
These should be `Debug` IMHO. Also you don't have a #3 log entry,. It goes 1,2,4,5. Would do away with the numbers.